.eslintrc.yml

```diff
@@ -17,23 +17,14 @@ parserOptions:
 plugins:
   - babel
 rules:
-  array-bracket-spacing: error
-  arrow-parens: error
   babel/new-cap: error
-  babel/object-curly-spacing: error
-  comma-dangle: [error, never]
   complexity: error
   eqeqeq: [error, always]
-  indent: [error, 2, {SwitchCase: 1}]
-  linebreak-style: [error, unix]
   no-await-in-loop: error
   no-negated-condition: error
   no-nested-ternary: error
   no-plusplus: error
   no-shadow: error
-  no-trailing-spaces: error
   no-unneeded-ternary: [error, {defaultAssignment: false}]
   no-var: error
   object-shorthand: [error, always]
-  quotes: [error, single]
-  semi: [error, never]
```

Comment:
removed some eslint rules that are purely stylistic and either clash with or are made redundant by prettier.